src/main/java/uk/gov/hmcts/juror/api/moj/report/standard/PersonAttendingSummaryReport.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import com.querydsl.core.Tuple;
+import com.querydsl.jpa.impl.JPAQuery;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.domain.IJurorStatus;
+import uk.gov.hmcts.juror.api.moj.domain.QJurorPool;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReport;
+import uk.gov.hmcts.juror.api.moj.report.DataType;
+import uk.gov.hmcts.juror.api.moj.repository.CourtLocationRepository;
+import uk.gov.hmcts.juror.api.moj.repository.PoolRequestRepository;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+
+@Component
+@SuppressWarnings("PMD.LawOfDemeter")
+public class PersonAttendingSummaryReport extends AbstractStandardReport {
+    private final CourtLocationRepository courtLocationRepository;
+
+    @Autowired
+    public PersonAttendingSummaryReport(PoolRequestRepository poolRequestRepository,
+                                        CourtLocationRepository courtLocationRepository) {
+        super(poolRequestRepository,
+            QJurorPool.jurorPool,
+            DataType.JUROR_NUMBER,
+            DataType.FIRST_NAME,
+            DataType.LAST_NAME);
+
+        this.courtLocationRepository = courtLocationRepository;
+        isCourtUserOnly();
+        addAuthenticationConsumer(request -> checkOwnership(request.getLocCode(), false));
+    }
+
+    @Override
+    protected void preProcessQuery(JPAQuery<Tuple> query, StandardReportRequest request) {
+        query.where(QJurorPool.jurorPool.nextDate.eq(request.getDate()));
+        query.where(QJurorPool.jurorPool.pool.courtLocation.locCode.eq(request.getLocCode()));
```

Comment:
if loc code comes from the court you are logged in as we should grab if from the SecurityUtil.getLocCode() instead of the request.

if owner based do the same but SecurityUtil.getActiveOwner()